src/main/java/core/basesyntax/service/accountingreport/Accounting.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service.accountingreport;
+
+import java.util.Map;
+
+public interface Accounting {
+    String COMMA = ",";
+    String FIRST_LINE = "fruit,quantity";
+    String accountingReport(Map<String, Integer> fruitKindsAndQuantity);
```

Comment:
Use Dependency injection in the impl class
```suggestion
    String makeReport();
```